code/datums/diseases/_disease.dm

```diff
@@ -1,7 +1,7 @@
 /datum/disease
 	//Flags
 	var/visibility_flags = 0
-	var/disease_flags = CURABLE|CAN_CARRY|CAN_RESIST
+	var/disease_flags = CURABLE|CAN_CARRY|CAN_RESIST|CHRONIC
```

Comment:
this cannot be a default flag. there are diseases that are subtypes of this that we don't want to randomly set to stage 1